.github/actions/build_pandas/action.yml

```diff
@@ -3,15 +3,28 @@ description: Rebuilds the C extensions and installs pandas
 runs:
   using: composite
   steps:
-
     - name: Environment Detail
       run: |
-        conda info
-        conda list
+        if which mamba 2>/dev/null; then
+          mamba info
+          mamba list
+          mamba info | grep -Ei 'environment.+:' | grep -qEiv 'environment.+:.+none'
+        fi
+        python -VV
+        which python
+        which pip
+        pip list
       shell: bash -el {0}
 
     - name: Build Pandas
       run: |
-        python setup.py build_ext -j 2
-        python -m pip install -e . --no-build-isolation --no-use-pep517 --no-index
+        time python setup.py build_ext -v -j $N_JOBS
+        pip install -v -e . --no-build-isolation --no-use-pep517 --no-index
```

Comment:
Why the `-v` and the `time`? Are you really missing something in the logs, or is it just a personal preference to add verbosity?

And why removing the `python -m pip` and calling just `pip`?

Would be useful if for such changes, instead of having them here were they are mostly unrelated to the main goal of this PR, you open PRs just with them, explaining your reasoning on why your proposal is better.

Reply:
`time`: It's hard to see how long things take from the GH UI but I added it mostly for debugging caching. Can remove.

I can leave `python -m pip` but it's being used inconsistently everywhere and I just wanted to make it consistent. Can do it in a follow-up PR. I think inconsistency with no good reason is very confusing.